src/main/java/com/hazelcast/eureka/one/EurekaOneDiscoveryStrategy.java

```diff
@@ -125,21 +134,29 @@ EurekaOneDiscoveryStrategy build() {
     private static final int DISCOVERY_RETRY_TIMEOUT = 1;
 
     private final EurekaClient eurekaClient;
+    private final String groupName;
     private final ApplicationInfoManager applicationInfoManager;
 
     private final Boolean useClasspathEurekaClientProps;
     private final String namespace;
     private StatusChangeStrategy statusChangeStrategy;
+    private final Boolean skipEurekaRegistrationVerification;
+    private final Boolean useMetadataForHostAndPort;
 
     private EurekaOneDiscoveryStrategy(final EurekaOneDiscoveryStrategyBuilder builder) {
         super(builder.logger, builder.properties);
 
         this.namespace = getOrDefault(EUREKA_ONE_SYSTEM_PREFIX, NAMESPACE, "hazelcast");
         boolean selfRegistration = getOrDefault(EUREKA_ONE_SYSTEM_PREFIX, SELF_REGISTRATION, true);
+        this.useMetadataForHostAndPort = getOrDefault(EUREKA_ONE_SYSTEM_PREFIX, USE_METADATA_FOR_HOST_AND_PORT, false);
+        this.skipEurekaRegistrationVerification = getOrDefault(EUREKA_ONE_SYSTEM_PREFIX, SKIP_EUREKA_REGISTRATION_VERIFICATION, false);
```

Comment:
Could you explain the purpose of this new property? Not sure I get it.

Reply:
This is useful option in my Spring Boot/Spring Cloud environment especially for testing (marked with `@VisibleForTesting` in `EurekaOneProperties`).

When first node starts, it takes some time to do self-registration with Eureka Server. Until Eureka data is updated it make no sense to verify registration (`EurekaOneDiscoveryStrategy.verifyEurekaRegistration()`). See [Time Lag](https://github.com/Netflix/eureka/wiki/Understanding-eureka-client-server-communication#time-lag). This option will speed up startup when testing.

Reply:
Do you think it would be useful in other scenarios, in production? I'm not keen on adding a new property just to facilitate tests. (For tests only, other mechanisms could be devised... perhaps through mocking?)

Reply:
Yes, this could be also useful in production when starting first node and if Eureka sync interval is longer then default 30s. 
For me this setting is useful for testing, when I start only one node with Hazelcast enabled. This is not related to unit tests.

Reply:
Sounds reasonable. Could you please add documentation for it as well in the README?

Reply:
Documented.